meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -1,3 +1,5 @@
+OVERRIDES =. "mender-enabled:"
```

Comment:
We are using `_mender-client-install` and other similar overrides in *a lot* of places already. Under what conditions exactly does it not work?

Assuming that that can be solved, I think we should not add the `mender-enabled` override, because everything should already fit into one of the variables added in `MENDER_FEATURES_ENABLE` and `MENDER_FEATURES_DISABLE`. Or else we should add more `MENDER_FEATURES` to cover the new cases, so that they can be enabled and disabled selectively.

Reply:
My only concern is, is it possible for mender-client-install _not_ to be set?

Reply:
Honestly, probably not. Would it be a problem though?

The intention to be able to disable it is there, and the idea was that this could be used if we added some product besides Mender that could be used independently (SWUpdate talking directly to Mender Server, CFEngine, etc.). Ultimately we may end up never using it for that purpose, but I still think it's better to use it than to introduce yet another one.

Reply:
OK.  I'll clean that up.

Reply:
Now I remember why I had to add that.  Since the Mender feature overrides are processed _later_ by anonymous python, they are usable for things such as the _impl bbclass selection.  I needed something that was always in the OVERRIDES list.

Reply:
I seem to recall having some issues with python in those variables not resolving correctly, but I'll give it a shot.

Reply:
I tested this, and it works.

Reply:
Yeah, I think it might have been when I was trying to use the _mender-* overrides that it didn't work.  It does seem to be working in my latest branch.